<launch>
    <node name="laplacian_line_detector" pkg="rr_iarrc" type="laplacian_line_detector" output="screen">
        <!--Minimum Area to Keep-->
        <param name="perfect_lines_min_cut" type="int" value="50" />
        <!--Strength of Floodfill-->
        <param name="Laplacian_threshold" type="int" value="60" />
        <!--Strength of Adaptive Thresholding-->
        <param name="adaptive_mean_threshold" type="int" value="3" />

        <!--The larger the height the lower it will be on the screen-->
        <param name="blockSky_height" type="int" value="95" />
        <param name="blockWheels_height" type="int" value="70" />
        <param name="blockBumper_height" type="int" value="0" />

        <param name="subscription_node" type="string" value="/camera/image_color_rect"/>
    </node>

    <include file="$(find rr_platform)/launch/sedani_state_publisher.launch"/>
    <include file="$(find rr_iarrc)/launch/lines_pcl_converter.launch"/>
</launch>
